src/Data/OsgNode.cpp

```diff
@@ -387,7 +387,7 @@ osg::ref_ptr<osg::Node> Data::OsgNode::createNodeSquare( const float& scaling, o
 	nodeQuad->setStateSet( bbState );
 
 	osg::ref_ptr<osg::Geode> geode = new osg::Geode;
-	geode->setNodeMask( geode->getNodeMask() & static_cast<unsigned int>( ~0x2 ) );
+    //geode->setNodeMask( geode->getNodeMask() & static_cast<unsigned int>( ~0x1 ) );
```

Comment:
treba alebo netreba ???
ak netreba zmazat,
ak mozno bude treba kvoli inym castiam kodu, tak to treba okomentovat